src/app.js

```diff
@@ -38,8 +37,8 @@ export default function (app) {
     }
 
     if (obj = plugin.hooks) {
-      Object.keys(obj).forEach(function (key) {
-        hooks[key].push(obj[key])
+      Object.keys(obj).map(function (key) {
```

Comment:
I generally prefer to use `map` over `forEach` whenever possible.. it feels much more pure as it has a return value, which also allows for chaining and AFAIK it is [actually faster](https://ryanpcmcquen.org/javascript/2015/10/25/map-vs-foreach-vs-for.html) than `forEach` in most cases so 👍 💯